packages/pre-install-always-fail/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "pre-install-always-fail",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

remove the error exit, we wont have a test here
```suggestion
    "test": "echo \"Error: no test specified\"",
```